.github/workflows/main.yml

```diff
@@ -9,8 +9,8 @@ on:
   workflow_dispatch:
 
 jobs:
-  tests:
-    name: Python ${{ matrix.python-version }}, ${{ matrix.os }}
+  check_python:
+    name: Check Python ${{ matrix.python-version }}, ${{ matrix.os }}
```

Comment:
I've gone for the name "check" here rather than "test", since we're also running black, mypy, flake8 etc.